packages/gatsby/src/commands/types.ts

```diff
@@ -20,6 +20,7 @@ export interface IProgram {
   https?: boolean
   sitePackageJson: PackageJson
   ssl?: ICert
+  prefixPaths: boolean
```

Comment:
I think this is wrong. I don't think prefixPaths is attached to all `IProgram`. I think it is only added in `gatsby serve`, which the interface is extended for. Let's revert

Reply:
Fixed in 1a65f05